pkg/plugins/resources/memory/store.go

```diff
@@ -151,6 +151,21 @@ func (c *memoryStore) Delete(_ context.Context, r model.Resource, fs ...store.De
 	}
 	return nil
 }
+func (c *memoryStore) DeleteMany(_ context.Context, fs ...store.DeleteManyOptionsFunc) error {
```

Comment:
I don't think it's a good idea to remove resources without taking into account their type